AdyenComponents/ACH Direct Debit/ACHDirectDebitComponent.swift

```diff
@@ -275,8 +275,10 @@ public final class ACHDirectDebitComponent: PaymentComponent,
             formViewController.append(storeDetailsItem)
         }
         
-        formViewController.append(FormSpacerItem(numberOfSpaces: 2))
-        formViewController.append(payButton)
+        if configuration.showsSubmitButton {
```

Comment:
Indeed, every component is responsible for creating and adding items to the form view. Unfortunately, with the current state it's hard to have a single point where this is done. Anyway, we can take a look into it. It would required a major refactor, but definitely up for it.

Reply:
Let's then have a session after I'm back from vacation?
I'm supercurious here as:
- even if we can't do it in one chunk, we can split this into reasonable chunks 
- this would help me to learn the codebase better. 

It does feel like these efforts could help us move faster in the future.